cmd/manager/main.go

```diff
@@ -50,9 +65,18 @@ func main() {
 	log.Info(fmt.Sprintf("KUDO Version: %s", fmt.Sprintf("%#v", version.Get())))
 
 	// create new controller-runtime manager
-	log.Info("setting up manager")
+	syncPeriod, err := parseSyncPeriod()
+	if err != nil {
```

Comment:
Let's go further and fail the manager in this case. Otherwise, errors related to wrongly formatted sync period values will be hard to debug.
```suggestion
	if err != nil {
		log.Error(err, "unable to parse manager sync period variable, run manager with defaults")
		os.Exit(1)
	}
```